webPages/webPages-war/src/main/webapp/webPages/jsp/editXMLContent.jsp

```diff
@@ -46,6 +46,8 @@
   context.setObjectId("0");
   context.setBorderPrinted(false);
   context.setShowMandatorySnippet(!otherFormDefined);
+  if (formUpdate.getData().isNew())
+    context.setCreation(true);
```

Comment:
Code rules: the `if` clause should starts with `{` and ends with `}` even whether it contains only a single statement